index.scss

```diff
@@ -256,24 +292,28 @@ $golden-scale: golden-scale(1);
 
   @if gr(no-flex-class) {
     #{unquote(gr(no-flex-class))} & {
-      > * {
-        @include column-padding;
-      }
-
       &:after {
-        content: '';
+        display: table;
```

Comment:
seems like there are reasons to go with `block`, thoughts? http://cssmojo.com/the-very-latest-clearfix-reloaded/